tools/Microsoft.StandardsPolice/project.json

```diff
@@ -1,6 +1,10 @@
 ﻿{
   "version": "1.0.0-*",
 
+  "compilationOptions": {
```

Comment:
Hmm this one shouldn't be needed I don't think. This is just a build tool, nothing compiles against it as far as I know.